lib/std/unicode.zig

```diff
@@ -196,22 +196,109 @@ pub fn utf8CountCodepoints(s: []const u8) !usize {
     return len;
 }
 
-pub fn utf8ValidateSlice(s: []const u8) bool {
+// Ported from go, which is licensed under a BSD-3 license.
+// https://golang.org/LICENSE
+//
+// https://golang.org/src/unicode/utf8/utf8.go
```

Comment:
It is not necessary to put a license statement for code ported (as opposed to copied) from another open source project, especially one that is BSD-3-clause licensed.

I don't even want it to mention that it is initially ported from Go, because that will lead contributors to assume the code is intended to track that other project as an upstream, rather than evolve and mutate within the Zig project to whatever makes sense for us.

Please de-goify the comments in the implementation as well. Unicode has no such thing as a "rune". It's called a "codepoint".